### PR TITLE
Add Menu Overlay Animations

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Adds Menu Overlay animations
- NOTE: These will be adjusted during QA with XXIX. I'm just getting the basic animation down :). 

### Type(s) of changes

- [x] New feature
- [x] Update to an existing feature

### How Has This Been Tested?

View at `/`
### Follow-up PR

- NOTE: These will be adjusted during QA with XXIX. I'm just getting the basic animation down :).

### CODE REVIEW

File: src/components/MenuOverlay.tsx [hunk 1]
```diff
@@ -29,50 +28,91 @@ const MenuOverlay: React.FC<Props> = ({ globalSettings, isOpen, mediaQuery }) =>
...
+        <div
+          className={cx('MenuOverlay__section-two-inner-container opacity-0 flex flex-col', {
+            'MenuOverlay__section-two-inner-container--active': isOpen,
+          })}
+        >
+          {menu.links.map((link: IButton) => (
+            <Button
+              key={`MenuOverlay-${link.link}`}
+              wrap={true}
+              className={cx(
```
Comment (mokaymokay) [inline src/components/MenuOverlay.tsx:86]: don't need cx here

**Review by mokaymokay — COMMENTED** [review]
the preview isn't working!

**Comment by chrismekim** [comment]
@mokaymokay it's working now! https://peek.run/g5wh0jm3

File: src/components/MenuOverlay.tsx [hunk 2]
```diff
@@ -29,50 +28,89 @@ const MenuOverlay: React.FC<Props> = ({ globalSettings, isOpen, mediaQuery }) =>
...
         {
           'events-none opacity-0': !isOpen,
           'events-all opacity-1': isOpen,
         }
       )}
     >
-      <div className="MenuOverlay__section-one bg-color-lilac-darker lg:col-6 flex flex-col lg:px3_75 lg:pb3_75 lg:pt15">
+      <div
+        className={cx(
+          'MenuOverlay__section-one bg-color-lilac-darker lg:col-6 col-12 flex flex-col lg:px3_75 lg:pb3_75 lg:pt15 transition-medium',
```
Comment (mokaymokay) [inline src/components/MenuOverlay.tsx:40]: is this meant to be `pt1_5`?

**Review by mokaymokay — APPROVED** [review]

File: src/components/MenuOverlay.tsx (hunk 2 above)
Comment (chrismekim) [reply src/components/MenuOverlay.tsx:40]: Haha no it's actually 15, their largest spacing unit haha. 😂 But thanks for looking out!!

**Pull request merged**